server/sonar-server/src/main/java/org/sonar/server/setting/ws/SetAction.java

```diff
@@ -73,16 +74,18 @@
   private static final Collector<CharSequence, ?, String> COMMA_JOINER = Collectors.joining(",");
 
   private final PropertyDefinitions propertyDefinitions;
+  private final I18n i18n;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "i18n" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1068)